scripts/playlists.py

```diff
@@ -0,0 +1,60 @@
+import locale
+import sys
```

Comment:
It might make sense to merge this file with `basic_youtube_upload.py` so you’ve just got one module where all the YouTube API stuff happens.